package/base-files/files/lib/upgrade/common.sh

```diff
@@ -101,7 +101,7 @@ get_image() { # <source> [ <command> ]
 		esac
 	fi
 
-	$cmd <"$from"
+	busybox $cmd <"$from"
```

Comment:
If you only want to alter a specific command, this should be changed in the switch case above instead.

Reply:
The `cat` command is originally provided by busybox, and busybox also provides the `bzcat` command. `bzcat` may also cause the same problems because of the installation of `bzip`, so adding the busybox prefix here can fix these problems together.